src/app/globals.css

```diff
@@ -24,7 +24,15 @@ body {
   align-items: center;
   justify-content: center;
 }
+.flex {
```

Comment:
`flex` is already available with Tailwind you shouldn't have to add a brand new class for that